ABDM_Consent_Management/.gitignore

```diff
@@ -0,0 +1,37 @@
+HELP.md
```

Comment:
Lets add .metadata as well to this list, we do not want to check in these changes